src/bentoml/server.py

```diff
@@ -184,11 +192,50 @@ def __exit__(
 
         return _Manager()
 
-    @abstractmethod
     def get_client(self) -> Client | None:
```

Comment:
@sauyon do you think that this is technically a breaking change?

Reply:
I don't believe so; it should only make things that didn't work before work now, right?

Reply:
I guess only case it breaks is user implementing their own server. But since we never expose it in docs anywhere i think it should be ok